GVFS/GVFS.Platform.Windows/ActiveEnumeration.cs

```diff
@@ -6,10 +6,11 @@ namespace GVFS.Platform.Windows
 {
     public class ActiveEnumeration
     {
-        private static FileNamePatternMatcher doesPatternMatch = null;
+        private static FileNamePatternMatcher wildcardPatternMatcher = null;
 
         // Use our own enumerator to avoid having to dispose anything
         private ProjectedFileInfoEnumerator fileInfoEnumerator;
+        private FileNamePatternMatcher patternMatcher;

```

Comment:
Rename these to better names:

patternMatcher -> doesPatternMatch
wildcardPatternMatcher -> doesWildcardPatternMatch